helm/prometheus-rules/templates/recording-rules/grafana-cloud.rules.yml

```diff
@@ -550,3 +550,7 @@ spec:
     rules:
       - expr: sum(trivy_compliance_info{title!~".*Pod.*"}) by (cluster_id, cluster_type, customer, installation, pipeline, provider, region, compliance_id, compliance_name, severity, title, status)
         record: aggregation:cluster_compliance_info
+  - name: capi_crds.grafana-cloud.recording
```

Comment:
Not blocking but could you document why you are adding this?

Reply:
I added a small comment but it seems kind of redundant. We just want to keep track of what crd versions are installed :shrug: